local/include/core/ProxyVerifier.h

```diff
@@ -864,6 +868,8 @@ class Session
 private:
   virtual swoc::Rv<size_t>
   drain_body_internal(HttpHeader &hdr, Txn const &json_txn, swoc::TextView initial);
+
+private:
```

Comment:
Why another private?

Reply:
Thanks for the review.

I find it helpful to repeat the modifier to separate private (or public or whatever) member functions from private member variables. It was a standard practice on my previous team that I like.